corehq/apps/users/middleware.py

```diff
@@ -80,7 +80,8 @@ def process_view(self, request, view_func, view_args, view_kwargs):
         if (toggles.TWO_FACTOR_SUPERUSER_ROLLOUT.enabled(request.user.username)
                 and not request.couch_user.two_factor_disabled
                 and not request.user.is_verified()
-                and not request.path.startswith('/account/')):
+                and not request.path.startswith('/account/')
+                and not request.path.startswith('/accounts/')):
```

Comment:
err... lol. When the `2FA` urls were made they didn't follow the same pattern as the original account ones (which is why this worked in the first place).

I think this would be a good place to use a decorator on the views that shouldn't have superuser 2FA, instead of hard coding the urls here. I think we discussed this earlier too. Something like `@disable_superuser_twofactor`, which would set an attribute on the view function which you could check here instead. 

This change also seems like it is too broad for the feature requested. URLs that start with `accounts` also include things like changing your password, which you shouldn't be allowed to do without enabling 2FA.

Reply:
Ohh, I thought that decorator logic was part of the PR that was getting QA'd, so I figured this would be a temporary fix until then (which was why I didn't get rid of the request.path.startswith stuff in my first PR of this stuff). Is that not the case? If so, Ill make it for this pr

Reply:
The decorator in that PR does something else (disables 2-FA entirely). This decorator would follow the same pattern, but would only disable the superuser 2FA thing.

Reply:
But the fix on the PR that preceded this was to allow 2FA to be enforced for all users (not just superusers and staff)

Do you mean there should be a decorator that specifically disables that feature flag on certain views?

Reply:
Ah, I think I see what you mean. I suppose you could use the same decorator and check for it here, is that what you are proposing? i think that makes sense. However, do you know why this check happens in middleware? Or why the other 2FA check doesn't happen in middleware? Maybe there is a way we can consolidate both?

Reply:
Consolidating sounds reasonable, I'll look into that when I have time / am not on interrupt anymore

Reply:
Oh yeah, should I wait for your changes to be merged in so I can use that decorator without causing too many conflicts?